integration-tests/common/gauntlet_common.go

```diff
@@ -33,54 +32,60 @@ func (testState *Test) fundNodes() ([]string, error) {
 		return nil, err
 	}
 
-	if testState.Common.Testnet {
+	if m.Common.TestEnvDetails.Testnet {
 		for _, key := range nAccounts {
 			// We are not deploying in parallel here due to testnet limitations (429 too many requests)
 			l.Debug().Msg(fmt.Sprintf("Funding node with address: %s", key))
-			_, err = testState.Sg.TransferToken(ethAddressGoerli, key, "100000000000000000") // Transferring 1 ETH to each node
+			_, err = m.Clients.GauntletClient.TransferToken(ethAddressGoerli, key, "100000000000000000") // Transferring 1 ETH to each node
 			if err != nil {
 				return nil, err
 			}
 		}
-
 	} else {
-		err = testState.Devnet.FundAccounts(nAccounts)
-		if err != nil {
-			return nil, err
+		// The starknet provided mint method does not work so we send a req directly
+		for _, key := range nAccounts {
+			res, err := m.TestConfig.Resty.R().SetBody(map[string]any{
+				"address": key,
+				"amount":  900000000000000000,
+			}).Post("/mint")
+			m.TestConfig.L.Info().Msg(fmt.Sprintf("Funding account: %s", string(res.Body())))
+			if err != nil {
+				return nil, err
+			}
```

Comment:
Needs to also mint STRK now, but should be resolved after you rebase

Reply:
Accepted remote changes